PatternPal/PatternPal/Services/LoggingService.cs

```diff
@@ -136,6 +137,21 @@ private static LogRequest CompileErrorLog(LogEventRequest receivedRequest)
         return sendLog;
     }
 
+    /// <summary>
+    /// Creates a LogRequest that is populated with info obtained from the supplied
+    /// received event and further specific details relevant for the FileEdit-event.
+    /// </summary>
+    /// <param name="receivedRequest">The originally received request from the PP extension</param>
+    /// <returns>A LogRequest populated for this specific event</returns>
+    private static LogRequest FileCreateLog(LogEventRequest receivedRequest)
```

Comment:
Discussion: This will not send along data, which can be considered fair since the file is empty and we could reason that it will be saved whenever the file is saved, but i'm unsure if I like this behaviour. @ovda96 would you weigh in your opinion?

Reply:
Just checked this, the file can have content from the template and in that case I think preferable behaviour would be to send this content along.

Reply:
Will be done in a different PR/branch.